azure-pipelines.yml

```diff
@@ -8,12 +9,12 @@ jobs:
 
     strategy:
       matrix:
-        Linux_Go113:
+        Linux_Go115:
           vm.image: 'ubuntu-18.04'
-          go.version: '1.13'
-        Linux_Go114:
+          go.version: '1.15'
```

Comment:
Now that 1.17 has shipped, can we change these versions to 1.16.7 and 1.17 respectively?

Reply:
Yes, that should be possible; do you want to fully "drop" 1.15, or keep 1.15 running in CI as well for some time as minimum version (while people upgrade)? Looks like CI is fairly short, and they run in parallel, so probably should be ok.

Reply:
I'm ok to drop it given that Google only supports the latest two versions.  However, CI is short as you pointed out, so if you have a compelling reason to keep it I'm fine with that.

Reply:
We can leave the `go.mod` changes at `1.15` though.

Reply:
If `go.mod` sticks at 1.15, it's probably good to also test on that version (just incase). Go 1.17 is pretty "fresh", and some project may linger a bit. Let me add 1.17 to the matrix 👍